lib/rihanna.ex

```diff
@@ -141,7 +151,13 @@ defmodule Rihanna do
       Rihanna.schedule(MyApp.MyJob, [arg1, arg2], in: :timer.minutes(5))
 
   """
-  @spec schedule(module, any, schedule_options) :: {:ok, Rihanna.Job.t()}
+  @spec schedule({module(), any()}, schedule_options(), map()) :: {:ok, Rihanna.Job.t()}
+  def schedule(term = {mod, _arg}, schedule_options, opts) when is_atom(mod) do
```

Comment:
This breaks the existing interface. All existing calls must continue to work after this change.

Reply:
This will not be required anymore after switching to a single options list. I'm deleting it